src/erasure-code/lrc/ErasureCodeLrc.cc

```diff
@@ -324,7 +324,7 @@ int ErasureCodeLrc::parse_kml(ErasureCodeProfile &profile,
     }
   }
 
-  if ((k + m) % l) {
+  if (l == 0 || (k + m) % l) {
```

Comment:
i could define a new error code for it, but the error message also applies to this case, i guess.